### PR TITLE
#11/UI/contest-component

### PR DESCRIPTION
**Pull request by MonikaKrella**
-created and added contest component into Contests Page
-after click color of component changes
-name, date, place, amount of competitors are taken by id from dummy-data 
-time is created in component (no dummy data for time)
-input labels are different elements from different task -> one of them changes colour with contest component (TODO to fix) and the second is showind date and state, which make it is quite long
-if you uncomment commented code navigation will work and send to page with same id as competition's id

image

image

-additionally there is also a file with function which creates array of dummy results in one class and for each dogs - it is out of task, could be removed, or if you want - could stay

AFTER COMMENTS:
-changed inappropriate letters in some words
-changed border size
-removed unnecessary css rules
-still impossible for me to change file name from contestComponent to ContestComponent

### CODE REVIEW

File: src/Molecules/ContestComponent/contestComponent.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import RANDOM_CONTESTS from '../../Data/Dummy-data/test-data-random-contests';
+import propTypes from 'prop-types';
+import InfoLabel from '../../Atoms/InfoLabel/InfoLabel';
+import {
+  ContestComponentStyled,
+  ContestNameStyled,
+  ContestInsideElementStyled,
+} from './ContestComponenyStyled';
```
Comment (RobertS-ki) [inline src/Molecules/ContestComponent/contestComponent.js:8]: Typo in file name (Componeny -> Component).

**Review by RobertS-ki — CHANGES_REQUESTED** [review]
General UI comments:
This is how it renders on our device:
obraz
css:
"cursor: default" could be used no to suggest editing possibility on the component
width of the info labels has to increase to fit all the text in one line (different issue/PR)
border width to be 1px around the ContestComponent

File: src/Molecules/ContestComponent/ContestComponenyStyled.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+import FONTS from '../../Styles/fontsStyledComponents';
+import COLORS from '../../Styles/varsStyledComponents';
+
+const ContestComponentStyled = styled.div`
+  box-sizing: border-box;
```
Comment (KatarzynaChmielecka) [inline src/Molecules/ContestComponent/ContestComponenyStyled.js:6]: Now we have box-sizing ing global styles, you can remove it :)

Comment (MonikaKrella) [reply src/Molecules/ContestComponent/ContestComponenyStyled.js]: done in new commit!

**Review by RobertS-ki — APPROVED** [review]

**Pull request merged**